### DISCUSSION TITLE
Import Option

### DISCUSSION THREAD

**Discussion by kkawatra09** [transcend-io/terragrunt-atlantis-config#347] [Q&A] [answered]
Hello, 
I'm wondering if this tool does a terragrunt import or has an option "atlantis import" like plan and apply. And it is not clear once we run the generate config command, how would that config look like, with other terragrunt options like state rm, taint, etc.

**Comment by Almenon** [comment, marked as answer]
You can run `terragrunt import` manually or use a import block if you're using Terraform v1.5.0 or later. 

https://developer.hashicorp.com/terraform/language/import

> And it is not clear once we run the generate config command, how would that config look like

Yeah, that could be improved. It's a open issue - see https://github.com/transcend-io/terragrunt-atlantis-config/issues/289
